lib/membrane_mp4/container/header.ex

```diff
@@ -25,12 +26,30 @@ defmodule Membrane.MP4.Container.Header do
   """
   @spec parse(binary()) :: {:ok, t, leftover :: binary()} | {:error, :not_enough_data}
   def parse(
-        <<size::integer-size(@size_size)-unit(8), name::binary-size(@name_size), rest::binary>>
+        <<compact_size::integer-size(@compact_size_size)-unit(8), name::binary-size(@name_size),
+          rest::binary>>
       ) do
+    {size, rest} =
```

Comment:
I think it would be cleaner to return `{header_size, content_size, rest}` here

Reply:
indeed, done